src/java.base/share/classes/jdk/internal/util/PlatformProps.java.template

```diff
@@ -55,7 +55,7 @@ class PlatformProps {
     private static String mapArchString(String archName) {
         return switch (archName) {
             case "x86_64" -> "x64";
-            case "ppc64le" -> "ppc64";
+            case "ppc64", "ppc64le" -> "ppc64";
```

Comment:
There’s no `case "s390" -> "s390"`, so this is also extraneous:
```suggestion
            case "ppc64le" -> "ppc64";
```

--------------------------------------------------------------------------------

The `case "ppc64"` was supposed to be added to `ArchTest.java`.